srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Hooks/User/User.php

```diff
@@ -400,14 +400,69 @@ function save_extra_user_profile_fields( $user_id ) {
         $accMgr->flush();
     }
 
-    if (false) {
-        $accessEntity = new AccountAccessEntity();
-        $accessEntity->setId($gameUser->getId());
-        $accessEntity->setGmLevel(0);
-        $accessEntity->setRealmID(-1);
+    // if (false) {
+    //     $accessEntity = new AccountAccessEntity();
+    //     $accessEntity->setId($gameUser->getId());
+    //     $accessEntity->setGmLevel(0);
+    //     $accessEntity->setRealmID(-1);
 
-        $accMgr->persist($accessEntity);
-        $accMgr->flush();
-    }
+    //     $accMgr->persist($accessEntity);
+    //     $accMgr->flush();
+    // }
 }
 
+function login_checks() {
+    ?>
+    <script>
+        function errorFactory(id, parent) {
+            const elemError = document.createElement("p");
+                elemError.style.color = "red";
+                elemError.id = id;
+                parent.appendChild(elemError)
+        }
+
+        function checkError(fieldLength, id, errorText) {
+            let error = "";
+            if (fieldLength > 16) {
```

Comment:
Can we not use the variables defined in UserValidator instead of hard coding?
Should we add min length check here? Regex check of characters used?

Reply:
you're right I will print the USERNAME and PASSWORD length from our conf variables

for the character user, I can add the check, anyway by default the user does not write his password to register but it is generated by the random password generator function, the password check it's only for who add the possibility to register with password and without the email check (I use this setting for testing purpose)